### PR TITLE
Use Python 3 for package.json filtering

### PR DESCRIPTION
**Pull request by evgeni**
Also drop old Python dev dependencies now that we don't vendor wesockify anymore

### CODE REVIEW

File: debian/bullseye/foreman/rules [hunk 1]
```diff
@@ -34,7 +34,7 @@ build:
 	/bin/sed -i '2i  gem "puma-status"' bundler.d/service.rb
 	$(BUNDLE) install
 	$(BUNDLE) package --no-install
-	/usr/bin/python script/filter-package-json.py
+	/usr/bin/python3 script/filter-package-json.py
```
Comment (ekohl) [inline debian/bullseye/foreman/rules:37]: It has a shebang. Should we rely on that? And perhaps long term also rewrite it to Ruby so we can drop Python as a build dependency?

**Review by ekohl — APPROVED** [review]
I wondered why we need it. If it's just a single script, perhaps we can rewrite it to Ruby.

File: debian/bullseye/foreman/rules (hunk 1 above)
Comment (evgeni) [reply debian/bullseye/foreman/rules:37]: Ah, that's a fair point, I think the explicit `/usr/bin/python` was to *override* the shebang… Let's try that.

Comment (evgeni) [reply debian/bullseye/foreman/rules:37]: Heh. The file is not executable. So won't work just now. I'll leave it with the explicit python3 and we'll revisit once we rewrite.

**Pull request merged**